Composer/packages/client/src/components/TriggerCreationModal/schema/defaultTriggerOrder.ts

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+export const defaultTriggerOrder = [
+  'Intent recognized',
+  'QnA Intent recognized',
+  'Unknown intent',
+  'Dialog events',
+  'Activities',
+  'Duplicated intents recognized',
+  'Custom events',
```

Comment:
Will this work if the strings are translated?

Reply:
No, it won't. We need a solution on that.

Reply:
I suggest that we add an extra `order` property in the UI Schema of `trigger` and `menu`. Action flyout menu is facing the same problem, we can't control the order of menu items.